app/models/specialist_document.rb

```diff
@@ -1,10 +1,12 @@
 class SpecialistDocument < ContentItem
-  attr_reader :continuation_link, :headers, :protection_type, :will_continue_on
+  attr_reader :continuation_link, :headers, :metadata,
+              :protection_type, :will_continue_on
 
   def initialize(content_store_response)
     super(content_store_response)
 
     @continuation_link = content_store_hash.dig("details", "metadata", "continuation_link")
```

Comment:
Note - content_store_hash may or may not exist depending on if this is merged before or after the PfG scaffolding PR (Nothing necessarily to do here, just need to note that these two PRs will clash one way or the other, and we have to bear in mind that some rebasing will be needed)